specs/2.0.0/index.html

```diff
@@ -3443,10 +3443,10 @@ <h2 class="no-toc">Table of contents</h2>
 
     <h3><a name="TIMEOUT_IGNORED">TIMEOUT_IGNORED</a></h3>
     <p>
-        In the WebGL 2.0 API TIMEOUT_IGNORED is defined as a GLint64 with the value -1 instead of a GLuint64
-        with the value 0xFFFFFFFFFFFFFFFF. This is because Javascript cannot accurately represent an integer
-        that large. For the same reason clientWaitSync and waitSync both take GLint64 values instead of GLuint64
-        for timeout.
+        In the WebGL 2.0 API <code>TIMEOUT_IGNORED</code> is defined as a <code>GLint64</code> with the value
+        <Code>-1</code> instead of a <code>GLuint64</code> with the value <code>0xFFFFFFFFFFFFFFFF</code>. This
```

Comment:
First tag: Code -> code. Here and below.

Reply:
Done